README.md

```diff
@@ -2,68 +2,58 @@
-- [ ] Actvitiy 4 - Connecting to a database and run a load test
-- [ ] Actvitiy 5 - Loading data from a CSV file and using that data in a HTTP Request
-- [ ] Activity 6 - A demonstration of how the HTTP Link parser works
-- [ ] Activity 7 - A Spidering example using the Selenium website
-- [ ] Activity 8 - To run a basic load test on the Selenium site for Training Support
-
-**JMeter Project:**
-- [ ] Activity 1 - Recording a load test
-- [ ] Activity 2 - Load testing an API with Authentication
-- [ ] Activity 3 - Petstore API
-
-**Linux:**
-- [ ] Activity 4: Create a shell script that creates 6 file each of types .png, .mp3, and .mp4. Move these to Image, Songs, and Movies folders respectively.
-
-**Docker:**
-- [ ] Activity 1 - Create and run a docker container that has a NodeJS app inside it
-- [ ] Activity 2 - Create and run a NodeJS app along with a DB container using docker networks
-- [ ] Activity 3 - Create and run a NodeJS app along with a persistently stored DB container using docker-compose
-- [ ] Activity 4 - Create and run a complex application that has multiple services
-
-**Kubernetes:**
-- [ ] Activity 1 - Create a configuration file to setup and run a simple Kubernetes cluster
-- [ ] Activity 2 - Create a configuration file to setup and run an application with multiple services running in a cluster
-- [ ] Activity 3 - Create a configuration file to setup and run a replicaset for an nginx pod
-- [ ] Activity 4 - Create a configuration file to setup and run an application with multiple services and persistent storage running in a cluster
-
-**Ansible:**
-- [ ] Activity 1 - Write a simple Ansible playbook
-- [ ] Activity 2 - Write an Ansible playbook to configure an application server
-
-**Docker Project:**
+JMeter:
+- [ ] Activity 1(JMX File) - To run a basic load test to demonstarate the basic elements in JMeter - Thread Group, Sampler, Assertions, Listener, Timers
+- [ ] Activity 2(JMX File) - To do a basic load test on a website using Basic Authentication
+- [ ] Activity 3(JMX File) - To implement basic load test using pre and post processors on the training support Selenium site
+- [ ] Actvitiy 4(JMX File) - Connecting to a database and run a load test
+- [ ] Actvitiy 5(JMX File) - Loading data from a CSV file and using that data in a HTTP Request
+- [ ] Activity 6(JMX File) - A demonstration of how the HTTP Link parser works
+- [ ] Activity 7(JMX File) - A Spidering example using the Selenium website
+- [ ] Activity 8(JMX File) - To run a basic load test on the Selenium site for Training Support
+JMeter Project:
+- [ ] Activity 1(JMX File) - Recording a load test
+- [ ] Activity 2(JMX File) - Load testing an API with Authentication
+- [ ] Activity 3(JMX File) - Petstore API
+
+Linux:
+- [ ] Activity 4(Screenshot): Create a shell script that creates 6 file each of types .png, .mp3, and .mp4. Move these to Image, Songs, and Movies folders respectively.
```

Comment:
Shouldn't this be 4 activities?
In the sheet we have 4 activities listed and the first 3 are screenshots and the 4th one is a sh file they need to upload.